content/en/resources/website/.gitignore

```diff
@@ -0,0 +1,3 @@
+# latest release blog
+!.gitignore
+*
```

Comment:
I don't understand this change. What's happening here?

Reply:
This is not required any more.